src/gource.cpp

```diff
@@ -1725,6 +1725,12 @@ void Gource::logic(float t, float dt) {
             idle_time = 0.0;
         }
 
+        if(gGourceSettings.transitions.size() > 0 && commit.timestamp > gGourceSettings.transitions.front()) {
```

Comment:
Because Gource can be used interactively and you could seek the time zone back to before the transition, this should really iterate over the transitions to find the relevant transition rather than modify the setting.